examples/simple_dff/Makefile

```diff
@@ -0,0 +1,16 @@
+
+TOPLEVEL_LANG ?= verilog
+
+ifeq ($(TOPLEVEL_LANG),verilog)
+  VERILOG_SOURCES = $(shell pwd)/dff.sv
+else ifeq ($(TOPLEVEL_LANG),vhdl)
+  VHDL_SOURCES = $(shell pwd)/dff.vhdl
+endif
+
+MODULE = test_dff
+TOPLEVEL = dff
+
+COCOTB_HDL_TIMEPRECISION = 1us
+COCOTB_HDL_TIMEUNIT = 1us
```

Comment:
Was that needed for something other than Verilator? So we can remove that now for simplicity?

Reply:
Yes. Verialotr supports `timescale` now so can be removed.